CM-1438 initial commit for fes-check.sh

nevans4-ch opened the pull request:
Checking in fes-check.sh script in monitoring dir
Also 2 minor logging fixes in move-scan-files.sh

nevans4-ch committed:
CM-1438 initial commit for fes-check.sh

Checking in fes-check.sh script in monitoring dir
Also 2 minor logging fixes in move-scan-files.sh

JMetherell commented on fes-batch/monitoring/fes-check.sh:
```diff
@@ -0,0 +1,103 @@
+
+   exit_code=$?
+   if [ $exit_code -ne 0 ]; then
+      echo "Error generating fes check report " >> $OUTFILE
```
You could possibly get rid of the `>> $OUTFILE` on the echos and just do `printReport > $OUTFILE`

nevans4-ch replied:
Done, thanks.

nevans4-ch committed:
CM-1438 updates to fes-check.sh after review

Using printReport > $OUTFILE instead of directing individual echo
statements to file.
